homework.md

```diff
@@ -4,89 +4,89 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 _ 3 _ 6 _ 1 _ 10 \* 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+It is the language for describing the presentation of Web pages, including colors, layout, and fonts
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
 #h1 {
-
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+JavaScript is a text-based programming language used both on the client-side and server-side that allows you to make web pages interactive
```

Comment:
So here on your answer you were on the right track till you then included, "What JavaScript does" which is out the scope of the question plus where you even deviated way more far from your answer is when you said, 
>  allows you to make web pages interactive

# Key on this question:
- Is for you to identify what JavaScript is, is it an Object Oriented Programming Language, Interpreted Programing Language, Compiled Programming Language etc and not to explain what it does.